README.md

```diff
@@ -74,14 +74,17 @@ title_name = "My application"
 assets = "static"
 # Optional, this is the default
 build_std = "std,panic_unwind"
-# Optional, true by default. Will strip debug symbols from the resulting elf when enabled.
-strip = true
-# Optional, this is the default
-vita_strip_flags = ["-g"]
 # Optional, this is the default
 vita_make_fself_flags = ["-s"]
 # Optional, this is the default
 vita_mksfoex_flags = ["-d", "ATTRIBUTE2=12"]
+
+[package.metadata.vita.dev]
+# Strips symbols from the vita elf in dev profile. Optional, default is false
+strip_symbols = true
+[package.metadata.vita.release]
+# Strips symbols from the vita elf in release profile. Optional, default is true
+strip_symbols = true
```

Comment:
I wonder if it could be 

```
[package.metadata.vita.profile.dev]
strip_symbols = true
[package.metadata.vita.profile.release]
strip_symbols = true
```

Notice the extra `profile`. It would be longer and make the logic more complicated (needing a map and also more logic for the default), but it would be possible to use arbitrary profiles. This already cover 99% of current use-cases though so I think it's good enough as it is.

Reply:
Makes perfect sense, will do

Reply:
Done